README.md

```diff
@@ -40,4 +46,12 @@ argument. Current version of the script file should produce the hash value of
 While the Docker IPFS container is running, you can access its web-ui via `http://localhost:5001/webui`.  
 
 If you are running the desktop IPFS application, you can upload the file through its interface. The hash of the file
-will then be displayed there.
+will then be displayed there.  
+
+When the application is started without any program arguments, it will first try to fetch IPFS file hash stored in the
```

Comment:
Can we change the logic of starting the application, if we don't provide arguments connect to infura and ipfs-gateway and if we provide an argument: `local` or `docker` then connect to local nodes. 
This way user doesn't have to know additional info about starting docker to run the application correctly but if he wants local nodes he needs to set them explicitly.

Reply:
Sure, will do